### PR TITLE
Autodesk Fusion 360 - CAD for hardware startups

### PR DESCRIPTION
**Pull request by neilpanchal**
Excellent option if you're making less than $100k/year in revenue. Fusion 360 is more than sufficient for most hardware startups and later can upgrade to a commercial version or get the professional Autodesk Inventor CAD software.

### CODE REVIEW

**Comment by camisatx** [comment]
Don't forget to add the new section to the `Contents` section (line 17?). Also, looks like they are sorting sections based on alphabetical order, so the new section may need to be moved up.

File: README.md [hunk 1]
```diff
@@ -77,10 +77,13 @@ put your business on the path to transformative growth. Disrupt your industry wi
 
 - [Intercom for Early Stage](https://www.intercom.com/early-stage) - Intercom’s user-friendly products help you acquire, onboard, engage and support your customers in a personal way. Eligible companies get all of Intercom's pro products for a flat rate of $49 a month for up to one year.
 
+### Hardware Development
+- [SOLIDWORKS](https://www.solidworks.com/solution/business-segments/entrepreneurs-startups) - For early-stage hardware startups that meet the eligibility criteria, the SOLIDWORKS for Entrepreneurs program offers software, training, and co-marketing resources at no cost to help you succeed.
+- [Autodesk Fusion 360](https://www.autodesk.com/campaigns/fusion-360-for-hobbyists) - CAD software for startups and hobbyist. If your entitlement has been designated as “Startup, ” You may use the service if You are (a) a company, startup, or home-based business that generates less than $100,000 (or equivalent in other currency) per year from the total sale of goods or services, or (b) an individual using the service for personal non-commercial projects, hobbies, or personal learning.
```
Comment (dakshshah96) [inline README.md]:
```suggestion
- [Autodesk Fusion 360](https://www.autodesk.com/campaigns/fusion-360-for-hobbyists) - Fusion 360 gives you access to everything you'll need in your product design and development process. As a startup or hobbyist user, you get professional 3D design, simulation, collaboration, and CNC machining software for free. Fusion 360 offers free use to qualifying startup and hobbyist makers through a simple 3-step activation process.
```

**Comment by neilpanchal** [comment]
Thanks, I just rearranged the order.

File: README.md [hunk 2]
```diff
@@ -63,6 +64,10 @@ put your business on the path to transformative growth. Disrupt your industry wi
...
 - [SendGrid Accelerate](https://sendgrid.com/accelerate/) - SendGrid Accelerate is a program to help startups succeed. Startups participating in the SendGrid Accelerate program receive 12 months of product credit, access to mentoring, and networking opportunities exclusive to program participants.
 
+### Hardware Development
+- [SOLIDWORKS](https://www.solidworks.com/solution/business-segments/entrepreneurs-startups) - For early-stage hardware startups that meet the eligibility criteria, the SOLIDWORKS for Entrepreneurs program offers software, training, and co-marketing resources at no cost to help you succeed.
+- [Autodesk Fusion 360](https://www.autodesk.com/campaigns/fusion-360-for-hobbyists) - CAD software for startups and hobbyist. If your entitlement has been designated as “Startup, ” You may use the service if You are (a) a company, startup, or home-based business that generates less than $100,000 (or equivalent in other currency) per year from the total sale of goods or services, or (b) an individual using the service for personal non-commercial projects, hobbies, or personal learning.
```
Comment (dakshshah96) [inline README.md]:
```suggestion
- [Autodesk Fusion 360](https://www.autodesk.com/campaigns/fusion-360-for-hobbyists) - Fusion 360 gives you access to everything you'll need in your product design and development process. As a startup or hobbyist user, you get professional 3D design, simulation, collaboration, and CNC machining software for free. Fusion 360 offers free use to qualifying startup and hobbyist makers through a simple 3-step activation process.
```

Comment (dakshshah96) [reply README.md]: Please update the description as suggested to match the one given on their website.

Comment (kylegiddens) [reply README.md]: [name](link) - Description.
[edgecase.ai] (www.edgecase.ai) - edgecase.ai provides a free $250 data annotation credit for any image or video based annotations for AI companies and projects. Additionally, researchers from academic institutions receive 2 months free of edgecase.ai's synthetic data generation platform [valid only with an EDU address].

**Comment by kylegiddens** [comment]
[edgecase.ai] ([www.edgecase.ai](http://www.edgecase.ai)) - edgecase.ai provides a free $250 data annotation credit for any image or video based annotations for AI companies and projects. Additionally, researchers from academic institutions receive 2 months free of edgecase.ai's synthetic data generation platform [valid only with an EDU address].

**Comment by dakshshah96** [comment]
@kylegiddens Please follow the [contribution guidelines](https://github.com/dakshshah96/awesome-startup-credits/blob/master/CONTRIBUTING.md) for suggesting new additions.

> Make an individual pull request for each suggestion.

**Pull request closed without merging**